.github/workflows/ci.yml

```diff
@@ -30,41 +42,50 @@ jobs:
       with:
         ref: ${{ github.event.pull_request.head.sha }}
 
+    # ~postgres/.pgx/config.toml comes in on the image.
+    # Setting HOME=~postgres (which is /home/postgres) in cargo's environment
+    # would work, but sticking with this approach for now.
```

Comment:
This does seem more elegant. I _think_ I ran into permissions issues when I tried to just run in the existing directory without changing ownership 🤔

Reply:
Here I'm only explaining what we're doing and why, not proposing a change.  The actions run as root, so if you had permission issues, you probably were running as postgres with data writable only by root.